source/wp-content/themes/wporg-developer/embed.php

```diff
@@ -38,13 +36,13 @@ function get_params( $post_id ) {
 /**
  * Returns a function string to display.
  *
- * @param int     $post_id
- * @param boolean $has_args
+ * @param int $post_id
  *
  * @return string
  */
-function get_signature( $post_id, $has_args ) {
+function get_signature( $post_id ) {
 	$title = get_the_title();
+	$has_args = count( get_params( $post_id ) ) > 0;
```

Comment:
@ryelle Why this change? I didn't want to make `get_params` a dependency of `get_signature` seeing that its values are not used. Although there are no tests, it does make it _less_ testable.

Reply:
Oops, I left my reasoning in the commit, not the comment — I did that so it would match the main theme's template tag (https://github.com/WordPress/wporg-developer/pull/59/commits/e52a950e31b70407c19ffbbfd2892dc4b70a6feb). It also surprised me that you would need to tell `get_signature` that the function has args, it seems like something that should be detected in the function (since usually it would also output those params - though I know it doesn't here).

Reply:
Got it. I think I would prefer to have it less similar than more. Maybe it makes more sense to create a truncated form of the function in `template-tags.php` since it does appear like we'll be keeping the signature around as it is with some modifications. 

What about something like:
```
function get_truncated_signature() {
 return get_signature( ..., true )
}

...

function get_signature( ..., truncate = false ) {

}
```

Reply:
Yeah, if we might use it elsewhere, I like your suggestion 👍🏻